docs/data/material/components/portal/portal.md

```diff
@@ -7,28 +7,13 @@ githubLabel: 'component: Portal'
 
 # Portal
 
-<p class="description">The portal component renders its children into a new "subtree" outside of current DOM hierarchy.</p>
+<p class="description">The Portal component lets you render its children into a DOM node that exists outside of its own DOM hierarchy.</p>
```

Comment:
It may be confusing what does the last "its" refer to - the Portal or the DOM node.

Reply:
That's a good point! I copied this text from the Base page, so I'll update that too.

Reply:
I just realized that it's not entirely clear to me which one is correct! 😅 As I understand, anything inside of the Portal is rendered in a new DOM node, meaning it exists outside of the Portal's DOM hierarchy—so "its" would refer to the Portal. Is that right?

Reply:
Yeah, this one is even more confusing 😅
It should refer to the Portal's DOM hierarchy, not the span's.

Reply:
Got it, I'll revise that text on the Base page to make that more clear.